lib/services/API/country_institute_api.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:mobile_app/constants.dart';
+import 'package:mobile_app/models/failure_model.dart';
+import 'package:mobile_app/utils/api_utils.dart';
+
+abstract class CountryInstituteAPI {
+  static const String COUNTRY = 'country';
```

Comment:
Not required if different functions are implemented. Also abstract classes should not define constants.

Reply:
I have moved the constants to HttpCountryInstituteAPI class . That should work right?